### PR TITLE
Add: subcourse.otherParticipants resolver

### PR DESCRIPTION
**Pull request by Leo310**
New access Role: COURSE_PARTICIPANT

### CODE REVIEW

File: graphql/subcourse/fields.ts [hunk 1]
```diff
@@ -151,6 +163,33 @@ export class ExtendedFieldsSubcourseResolver {
...
+                schooltype: true,
+            },
+        });
+    }
+
+    @FieldResolver((returns) => [OtherParticipant])
+    @Authorized(Role.SUBCOURSE_PARTICIPANT)
+    @LimitEstimated(100)
+    async otherParticipants(@Root() subcourse: Subcourse) {
+        return await prisma.pupil.findMany({
```
Comment (Jonasdoubleyou) [inline graphql/subcourse/fields.ts:180]: This should exclude the user itself (pupilId: { not: sessionPupil.id })

**Review by Jonasdoubleyou — APPROVED** [review]

**Comment by Leo310** [comment]
What happened in the integration test?

**Comment by Jonasdoubleyou** [comment]
Timing issue. The backend is started, then it waits 5s, then it runs the tests. Sometimes the backend takes too long to start up ... Need to fix this someday

**Comment by Leo310** [comment]
ah ok I see

**Pull request merged**